posts/2019/05/quansight-partners-with-quansight-to-sponsor-spyder.md

```diff
@@ -0,0 +1,112 @@
+copying from one project to another is done in the 'Explorer' by means of
+context menus. 
+
+It is worth mentioning that projects are considered an “Advanced feature”,
+so it will not be forced on users that just want to get some work done.
+Projects usage is, and will remain, and optional feature.
+
+<!-- TEASER_END -->
+
+## Project Types
+
+Since the revamp of Spyder Projects with the 3.x series, the new dialog hinted
+the possibility of creating different types of projects. Currently spyder
+provides an empty project type.
+
+![Project creation](/images/spyder-project-creation.png)
+ 
+This feature will:
+* Expand the (current) Project Explorer API to handle Project Types.
+* Define and implement the minimal aspects that should be extensible by a
+Project Type
+
+### File Associations
+
+Spyder currently supports opening files on the Project and File Explorer
+using the (default) Operating System defined applications. We will enhance
+this by adding **Global** file associations as part of the **Preferences**
+and Project Specific associations as part of the **Project Preferences**.
+This will give users more power to select the tool or set of tools that they
+prefer to use for specific projects and tasks directly on the File and Project
+Explorer pane.
+
+### URL Awareness
+
+Working with projects require access to documentation, located within the
+computer as other files or links located over the internet. We will now
+include integration for URI detection within the editor so users can
+<kbd>Ctrl+Click</kbd> on these items. This will work similarly as module
+inspection works, but generalized to external and internal links, which
+might include:
+
+### Git version control enhancements
+
+The revision system of choice by our user base, including TDK, is Git.
+The purpose of this enhancement is to enable basic Git integration from
+within the File and Project Explorer and to extend the current plugin
+API so context menus can be customized with specific Git features and
+commands.
+ 
+Initial work has already started to display Git relevant information
+on the status bar
+
+![Status bar git](/images/spyder-git-status.png)
+
+## Editor alternative views
+
+There are many circumstances where a user would like to edit a file that might
+be rendered using a parsing tool into another type of view. One example of
+this, is [markdown](), the language on which this blog post is written, which
+can be rendered to HTML. At the moment the only way to achieve this on Spyder
+is via the use of a dedicated plugin that would create a new type of Pane. And
+example of such plugin is [spyder-reports](https://github.com/spyder-ide/spyder-reports)
+(See image below and the Reports pane). The reason why plugins are developed
+this way is because the Editor Plugin on Spyder can only contain Code Editor
+widgets, which prevents us and other developers to use the Editor to hold
+these AlternateViews of a given file.
```

Comment:
better just use "alternative views" consistently as the term. this sounds like a class name